crates/common/src/types.rs

```diff
@@ -44,6 +44,26 @@ pub enum Chain {
     },
 }
 
+pub enum MainnetLidoModule {
+    Curated = 1,
+    SimpleDVT = 2,
+    CommunityStaking = 3
```

Comment:
```suggestion
    CommunityStaking = 3,
```

Reply:
`fmt-check` needs these commas here for CI to pass, Clippy doesn't care which is interesting; sorry about that!